.github/workflows/check-latest-images.yaml

```diff
@@ -0,0 +1,45 @@
+name: Update sample build strategy images.
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  check-new-versions:
+    name: Update sample build strategy images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Check and modify kaniko image version
+        env:
+          IMAGE: gcr.io/kaniko-project/executor
+          LATEST_RELEASE: https://api.github.com/repos/GoogleContainerTools/kaniko/releases/latest
+        run: |
+          for directory in samples test; do hack/check-latest-images.sh ${IMAGE} ${LATEST_RELEASE} ${directory}; done
+      - name: Check trivy image versions
+        env:
+          IMAGE: docker.io/aquasec/trivy
+          LATEST_RELEASE: https://api.github.com/repos/aquasecurity/trivy/releases/latest
+        run: |
+          for directory in samples; do hack/check-latest-images.sh ${IMAGE} ${LATEST_RELEASE} ${directory}; done
+      - name: Compare and modify buildah image version
+        env:
+          IMAGE: quay.io/containers/buildah
+          LATEST_RELEASE: https://api.github.com/repos/containers/buildah/releases/latest
+        run: |
+          for directory in samples test; do hack/check-latest-images.sh ${IMAGE} ${LATEST_RELEASE} ${directory}; done
```

Comment:
This does not work. The latest BuildAh release is 1.24.4. But, the image is not provided by the BuildAh team. I think this comes from Fedora. And there, the newest image is 1.23.3. I think for BuildAh, we will need some different logic.

Reply:
@SaschaSchwarze0 Thanks for your input and apologies for late response. Can you please share the link where I can track the correct Buildah release if it is not GitHub one?

Reply:
Hi @kevydotvinu, you need to look at the tags that are available in quay, https://quay.io/repository/containers/buildah?tab=tags&tag=latest. Atm the situation is that the latest GitHub release is 1.28.0 which the newest image is on 1.27.0.

Reply:
```bash
curl https://quay.io/api/v1/repository/containers/buildah/tag/ | jq -r '.tags | .[0].name'
```